homeassistant/components/switch/__init__.py

```diff
@@ -105,6 +109,16 @@ def today_power_mw(self):
         return None
 
     @property
+    def standby_state(self):
+        """ Is the device on - or in standby. """
+        return None
+
+    @property
+    def sensor_state(self):
+        """ Is the sensor on or off. """
```

Comment:
So I am not sure if I get correctly what this attribute means.

Is this the ability that it can sense if a device is in stand by? In that case we can have `standby_state` have three different return values:
 - `True` - device is in standby
 - `False` - device is not in standby
 - `None` - standby state is not supported